test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -971,6 +998,8 @@ public void testLogEntrySyncWithTrim() throws Exception {
         // Setup Environment: two corfu servers (source & destination)
         setupEnv();
 
+        log.info("Have setutEnv Done");
```

Comment:
required?

Reply:
When the test goes wrong, we know the process and where it stuck.